tests/topologyDescription/topologyDescription-hasReadableServer_error-001.phpt

```diff
@@ -44,7 +44,6 @@ class TopologyDescriptionProvider implements MongoDB\Driver\Monitoring\SDAMSubsc
 }
 
 $tests = [
-    null,
```

Comment:
I can add language to the documentation in PHPC-1710.

This is one of those special cases that can't be reproduced in PHP userland code as it doesn't know the concept of optional, non-nullable arguments. The only way to prevent users from passing an explicit `null` value would be to make the parameter required, which would be a BC break. I figured if users want to pass `null` there's nothing that would prevent us from handling it the same way as if they didn't pass anything at all.